### DISCUSSION TITLE
[1084600]沙石镇时光 {本体}

### DISCUSSION THREAD

**Discussion by hundouhuaer** [pjy612/SteamManifestCache#919] [Share]
### Steam 游戏ID

1084600

### 密钥列表

```hash
`appid:1084600`

1084601;7af65c0378d727ff872dcf1bfc511faa6bda56ddb31f4d891ba88a15e5540d17
```

### 清单附件

[1084601_245527738141462187.zip](https://github.com/user-attachments/files/18193720/1084601_245527738141462187.zip)

### 备注

hundouhuaer@gmail.com
顺带一问，我之前的内测版已经不能用了，最新版解锁每个游戏都是缺少密钥，是作者调整了吗。

**Comment by shirotoria** [comment]
确实，我也这样
